ci/jjb/jobs/docs.yaml

```diff
@@ -64,3 +64,114 @@
     publishers:
       - email-notify-owners
       - mark-node-offline
+
+
+# This set of Jenkins jobs which build the Pulp platform and plugin documentation and publishes
+# it to https://docs.pulpproject.org/ for Pulp3
+
+- job-template:
+    name: 'docs-builder-3.0-dev'
+    defaults: ci-workflow-runtest
+    node: 'fedora-np'
+    properties:
+        - docs-ownership
+    scm:
+        - git:
+            url: 'https://github.com/pulp/pulp-ci.git'
+            branches:
+                - origin/master
+            basedir: pulp-ci
+            skip-tag: true
+            wipe-workspace: false
+        - git:
+            url: https://github.com/pulp/devel.git
+            branches:
+              - '3.0-dev'
+            skip-tag: true
+            basedir: devel
+            wipe-workspace: false
+    triggers:
+        - timed: '{trigger_times}'
+    wrappers:
+        - ssh-agent-credentials:
+            users:
+                - '044c0620-d67e-4172-9814-dc49e443e7b6'
+                - '2f7a0c14-d520-40d3-b9b5-ebd8bb780d03'
+        - credentials-binding:
+            - zip-file:
+                credential-id: 9051da21-c8af-49bd-a0ac-c1dd94a6d216
+                variable: KOJI_CONFIG
+        - timeout:
+            # Timeout in minutes
+            timeout: 240
+            timeout-var: 'BUILD_TIMEOUT'
+            fail: true
+    builders:
+        - shell: |
+            #!/bin/bash
+            # install packages deps required to build docs
+            sudo dnf install graphviz plantuml python-virtualenv postgresql-devel python3-devel gcc ansible -y
```

Comment:
`python-virtualenv` should be unnecessary. See the comment on making a virtualenv, below.